components/support/nimbus-fml/src/defaults/merger.rs

```diff
@@ -174,6 +174,44 @@ impl<'object> DefaultsMerger<'object> {
         feature_def: &mut FeatureDef,
         defaults: &Option<Vec<DefaultBlock>>,
     ) -> Result<(), FMLError> {
+        let variable_defaults = self.collect_feature_defaults(feature_def)?;
```

Comment:
Are there pre-existing tests covering these changes?

Reply:
Oh yes! The tests covering this code start [here](https://github.com/mozilla/application-services/blob/2b05e22f2e2d5f2f8f4eb4573244061b91700240/components/support/nimbus-fml/src/defaults/merger.rs#L345).